.gitlab/fragments/pipeline.yaml

```diff
@@ -0,0 +1,3 @@
+variables:
```

Comment:
This is where the environment variables are set.

Reply:
@hannes-ucsc, @bvizzier-ucsc You need to do the equivalent of `export GATSBY_ENV="ANVIL-PROD"` before running the build.

See:

https://github.com/anvilproject/anvil-portal/blob/107c9c0ed1cf000a102e9bd1bca728a842cf57c8/src/utils/environment/environment.service.js

And
https://github.com/anvilproject/anvil-portal/blob/107c9c0ed1cf000a102e9bd1bca728a842cf57c8/src/utils/environment/environment-url.model.js

Then you can remove the clobbered line.

Cheers,
Dave

Reply:
I removed the commit.

Reply:
We are setting GATSBY_ENV in base.yaml below.

> Then you can remove the clobbered line.

That surprises me since the [clobbered line](https://github.com/anvilproject/anvil-portal/blob/107c9c0ed1cf000a102e9bd1bca728a842cf57c8/src/config/anvil/config.ts#L29) always appends `data` which wouldn't work in this case.

Reply:
But again, I removed the commit and will let you figure this out.